### DISCUSSION TITLE
Table组件中，导出PDF功能导出的PDF文件大小为0KB

### DISCUSSION THREAD

**Discussion by ya00ya11** [dotnetcore/BootstrapBlazor#3165] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

BB版本：8.3.9 
运行环境：win11
浏览器：Microsoft Edge

操作：
在主页的Table中，点击导出数据→Pdf格式

现象：
导出的PDF文件大小为0KB

能够复现该Bug的项目文件：
[BootstrapBlazorDemo.zip](https://github.com/dotnetcore/BootstrapBlazor/files/14798984/BootstrapBlazorDemo.zip)

### Expected Behavior

能够正确导出pdf文件

**Comment by bb-auto** [comment]
@ya00ya11 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@ya00ya11 视频教学链接 https://space.bilibili.com/660853738/search/video?keyword=%E5%AF%BC%E5%87%BA%20PDF

网站示例实现类

https://github.com/dotnetcore/BootstrapBlazor/blob/main/src/BootstrapBlazor.Server/Services/ExportPdfService.cs

请点一下下面的 mark as answer
